.github/CONTRIBUTING.md

```diff
@@ -84,6 +84,34 @@ Topics that are around understanding something can be written as a blog post (we
 
 ## Translations (i18n)
 
-While we plan to support translations in the future, we are not currently accepting them until Tauri 2.0 is a bit more stable.
+Thanks for your interest in helping to translate the documents! While we plan to support translations in the future, **we are not currently accepting them until Tauri 2.0 is a bit more stable**.
 
 Once translations are enabled you can check the [translation status page](https://beta.tauri.app/contribute/translate-status).
+
+To ensure the translation is consistent across multiple contributions, be sure to communicate with others contributors and consult with reliable external resources such as [Microsoft terminology search](https://msit.powerbi.com/view?r=eyJrIjoiODJmYjU4Y2YtM2M0ZC00YzYxLWE1YTktNzFjYmYxNTAxNjQ0IiwidCI6IjcyZjk4OGJmLTg2ZjEtNDFhZi05MWFiLTJkN2NkMDExZGI0NyIsImMiOjV9) during translation.
+
+### Enabling a language
+
+If you seek to contribute but find your language missing, please open an issue.
+
+### Starting a translation
+
+Before you start working on a translation, look through the [open pull requests](https://beta.tauri.app/contribute/translate-status#needs-review) to see if anyone else is already working on one for your language.
+
+If none spotted, then now is time for you to lead the effort! So here's how to start:
+
+1. Head to [translation status page](https://beta.tauri.app/contribute/translate-status).
+2. Look for missing translation in corresponding languages.
+3. Hit `Create page +` which will help you automagically fork the repository and create the corresponding page.
+4. Once your translation is done, initiate a pull request.
+
+Note that you may send send a pull request before your translation is complete: e.g., you may want to ask for help with translations. However, your pull request will not be merged until the translation is complete.
+
+### Updating a translation
+
+#### Corrections
+
+If you notice spelling or grammar errors, typos, or opportunities for better phrasing, open a pull request with your suggested fix. If you see a problem that you aren't sure of or don't have time to fix, open an issue.
+
+#### Broken links
+When broken links are found, try to fix them across all translations. Ideally, only update the linked URLs, so that translation changes will definitely not be necessary.
```

Comment:
I think we also need to provide some guidance for how to write links here. i.e. Absolute vs relative links (absolute is preferred), if you include the locale in a link (I believe so, but might need to double-check that behaviour), how to handle headings (such as `#link-to-heading`).

One extra thing I can look at is adding in a link validator (such as https://github.com/HiDeoo/starlight-links-validator) that we run. This is [talked about being added into Starlight natively in the future](https://github.com/withastro/starlight/issues/247) but I can add this specific integration now if we think we want. That HiDeoo guy is awesome and is very involved in the Starlight ecosystem

Reply:
Good point, though not sure if we're supposed to do this here in Translation Guide or in a separate Style Guide.

And yes, I was thinking we'd be adding an action to test on the links when I write up this line. Nice to hear there is already a potential working one and a native one in the future!

Reply:
For sure the styles guide (making a note to write one up). Maybe the locale-specific things we could include in the i18n guide and then also link out to the style guide as well?

Reply:
Makes sense to me! Let's do that.